src/Assertion/AssertionContainer.php

```diff
@@ -28,10 +28,16 @@
  *
  * @author  Aeneas Rekkas
  * @licence MIT
- *
- * @method AssertionInterface get($name)
  */
-class AssertionPluginManager extends AbstractPluginManager
+final class AssertionContainer extends AbstractPluginManager implements AssertionContainerInterface
 {
     protected $instanceOf = AssertionInterface::class;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function get($name, array $options = null): AssertionInterface
+    {
+        return parent::get($name, $options);
```

Comment:
Maybe better go safely here and don't pass $options here? since we don't have this in an interface

Reply:
another solution would be we could extend ServiceManager instead of PluginManager